source/content/guides/decoupled/drupal-backend-starters/02-create.md

```diff
@@ -77,7 +81,15 @@ Replace `'{My Team Name}'` with your team name. For example, `My Agency`. This c
 
 </Alert>
 
-## Install with Build Tools
+## Create with Terminus Plugin
+
+<Partial file="decoupled-terminus-plugin-backend.md" />
+
+## Create with Node.js CLI
```

Comment:
Since this is about creating a backend project, I don't think the Node CLI option belongs here. I think that would fit better in the create a new project section of each of the FE starters: /guides/decoupled/drupal-nextjs-frontend-starters/create